MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -30,6 +30,11 @@ public enum VisualInstructionComponentType: Int, CustomStringConvertible {
      */
     case exit
     
+    /**
+     The component contains information about which lanes can be used to complete the maneuver.
+     */
+    case lane
```

Comment:
This case becomes unnecessary because LaneIndicationComponent no longer has a `type` property.